lib/magma/model.rb

```diff
@@ -238,8 +249,16 @@ def inherited(magma_model)
         )
 
         super
-        magma_model.attribute(:created_at, type: DateTime, hide: true)
-        magma_model.attribute(:updated_at, type: DateTime, hide: true)
+        magma_model.send(:attribute, :created_at, type: DateTime, hide: true)
```

Comment:
could these instead use the new interface, e.g. magma_model.date_time(:created_at, hide:true)?